### ISSUE TITLE
units.parse_bytes should not panic when given an empty string

### ISSUE THREAD

**Issue by tsandall** [open-policy-agent/opa#2901]
The subject line tells the whole story!

```
OPA 0.25.0-dev (commit 3fb39b56, built at 2020-11-12T18:12:38Z)

Run 'help' to see a list of commands and check for updates.

> units.parse_bytes("")
```

**Comment by srenatus** [comment]
I believe this was fixed in #2902. 😉

**Comment by tsandall** [comment]
Yep, missed this in the commit message. Whoops!
